api/src/main/scala/scalalab3/model/Song.scala

```diff
@@ -0,0 +1,9 @@
+package scalalab3.model
```

Comment:
There is an inconsistency of package naming with the previous commit. We should all agree on some package convention.